File: playground_series/s3e4/py/baseline.py
import pandas as pd
import statistics

path = "./playground_series/s3e4/data/"

trn = pd.read_csv(path + 'train.csv')
tst = pd.read_csv(path + 'test.csv')

trn.head(1)

pred = statistics.mean(trn.Class)

data = {
    'id' : trn.id,
    'Class' : pred
}


sub = pd.DataFrame(data = data)

sub.to_csv("./playground_series/s3e4/submissions/baseline_py.csv")
    

File: playground_series/s3e4/jl/baseline.jl
using CSV
using DataFrames
using Statistics
using Parquet2

#read parquet
#note that I already converted these to parquet
trn = DataFrame(Parquet2.Dataset("./playground_series/s3e4/data/train.parquet"); copycols = false)
tst = DataFrame(Parquet2.Dataset("./playground_series/s3e4/data/test.parquet"); copycols = false)

names(trn)

pred = mean(trn.:Class)

sub = DataFrame(
    id = tst.:id,
    Class = pred
)

CSV.write("./playground_series/s3e4/submissions/baseline.csv", sub)